tests/Sort.test.ts

```diff
@@ -46,18 +47,32 @@ describe('Sort', () => {
             path: '',
         });
         expect(
-            Sort.by({ sorting: [{ property: 'due', reverse: false }] }, [
-                one,
-                two,
-                three,
-            ]),
+            Sort.by(
+                {
+                    sorting: [
+                        {
+                            property: 'due',
+                            reverse: false,
+                            propertyInstance: 0,
+                        },
+                    ],
+                },
+                [one, two, three],
```

Comment:
Thinking out loud, for future reference only (it's definitely not worth changing the code now...)

I wonder whether it might be possible in such cases in future to adjust the tests or the code to make the propertyInstance integer optional, to remove the need to update all the existing tests?

Reply:
Created a issue to track. #657